framework/dora_api/infrastructure/middleware.py

```diff
@@ -38,18 +39,42 @@ async def deserialise_web_request():
     _RequestEndpoint = request.endpoint.split(".")[-1]
     if _RequestEndpoint in REQUEST_BODYS_BY_ENDPOINT:
         _RequestData: dict = request.get_json()
+        _DeserialisedRequestData : dict = dict()
```

Comment:
Can do = {} as shorthand btw

Reply:
mind blown, that syntax makes me think object because of javascript